backend/.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   plugins: ['@typescript-eslint/eslint-plugin'],
   extends: [
     'plugin:@typescript-eslint/recommended',
-    'plugin:prettier/recommended',
+    //'plugin:prettier/recommended',
```

Comment:
Načeloma bi take stvari kar izbrisal, saj imamo sledljivost v gitu in ne vidim potrebe po zakomentiranju.